app/templates/events/view/tickets/access/create.hbs

```diff
@@ -0,0 +1,81 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="ui header column">
+      {{t 'Create an access code'}}
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <form class="ui form">
+        <div class="field">
+          <label class="required">{{t 'ACCESS CODE'}}</label>
```

Comment:
@sumedh123 like i have said so many times before, we don't want to _copy_ things directly from the server. The server is far from perfect. Should be used just for the wireframe